src/queryResult/queryResultWebViewController.ts

```diff
@@ -60,17 +72,105 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
             state.tabStates.resultPaneTab = payload.tabId;
             return state;
         });
+        this.registerReducer("getExecutionPlan", async (state, payload) => {
+            this._executionPlanContents = payload.sqlPlanContent;
+            await this.createExecutionPlanGraphs();
+            state.loadState = ApiStatus.Loaded;
+            state.tabStates.resultPaneTab =
+                qr.QueryResultPaneTabs.ExecutionPlan;
+            return {
+                ...state,
+                sqlPlanContent: this._executionPlanContents,
+                executionPlan: this.state.executionPlan,
+                executionPlanGraphs: this.state.executionPlanGraphs,
+            };
+        });
+        this.registerReducer("saveExecutionPlan", async (state, payload) => {
+            let folder = vscode.Uri.file(homedir());
+            if (await exists("Documents", folder)) {
+                folder = vscode.Uri.file(path.join(folder.path, "Documents"));
+            }
```

Comment:
.sqlplan files aren't documents, so let's not put them there.  I think homedir is fine as a default location.